docs/src/user/benchmark.rst

```diff
@@ -0,0 +1,30 @@
+**********
+Benchmark
+**********
```

Comment:
You should point to the jupiter notebook example in this doc.

Reply:
have not tried this before, but how can be put a git repo link in the doc so it will not affected by the `branch`, or just always point to `develop`?

Reply:
I would point to develop. At the time of release, we replace with master.